src/pick/__init__.py

```diff
@@ -110,8 +110,8 @@ def get_option_lines(self) -> List[str]:
 
         return lines
 
-    def get_lines(self) -> Tuple[List[str], int]:
-        title_lines = self.get_title_lines()
+    def get_lines(self, max_x: int = 80) -> Tuple[List[str], int]:
```

Comment:
I prefer `max_width` instead of `max_x` for better understand. And I prefer it as a keyword only parameter. `get_title_lines` should apply the same change.

```suggestion
    def get_lines(self, *, max_width: int = 80) -> Tuple[List[str], int]:
```